### DISCUSSION TITLE
Library cublasLt is not initialized

### DISCUSSION THREAD

**Discussion by stdwor** [zai-org/ChatGLM3#1222] [Q&A] [answered]
windows11 的环境，请问我运行的 model = AutoModel.from_pretrained(MODEL_PATH, trust_remote_code=True).quantize(4).cuda().eval() 的时候出现错误RuntimeError: Library cublasLt is not initialized
nvidia-smi运行可以查看到cuda版本为12.0，nvcc没有这个命令
这个问题RuntimeError: Library cublasLt is not initialized 可能的原因是什么。。

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
这个是库没装吧，nvcc都没有可能就没有读取到你的cuda，nvidia smi看到的是支持的最高cuda版本
